src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/Referral.kt

```diff
@@ -103,4 +100,10 @@ class Referral(
 
   val currentActionPlan: ActionPlan?
     get() = actionPlans?.lastOrNull()
+
+  val currentAssignment: ReferralAssignment?
+    get() = assignments.lastOrNull()
```

Comment:
Will `last` always return the latest? Perhaps might need to sort by assignedAt.

Reply:
good point - it _should_ since the list is always appended to, but can't hurt to make it explicity

Reply:
yeah. I was thinking of more when it's returned by a hibernate fetch and how it orders it. Normally it would be based on the natural order of the database (using `ctid`) but not 100% sure it will always be in the order of insertion (maybe if there's an update for instance).